src/screens/Settings/index.tsx

```diff
@@ -100,7 +118,7 @@ function Settings() {
     cloudSaveEnabled: false,
     saveFolder: ''
   })
-  const [autoSyncSaves, setAutoSyncSaves] = useState(false)
+  const [autoSyncSaves, setAutoSyncSaves] = useState(true)
```

Comment:
Is this safe to enable by default now?

Reply:
Yes, safe because we have another check on the game side. If the game has no cloud save then it won't check that.

Reply:
Actually I should revert this. Because we don't have the folder until the user goes to sync settings.

Reply:
Yeah I thought so too.

Reply:
This isn't fixed yet?

Reply:
Fixed.